test/etc/profile.definitions/site/buildtest.yml

```diff
@@ -0,0 +1,15 @@
+## help buildtest
+
+BUILDTEST:
+  BINARY:                          "False"
+  CLEAN_BUILD:                     "False"
+  EASYBUILD:                       "False"
+  OHPC:                            "False"
+  CONFIGS_REPO:                    $HOME/buildtest-framework/toolkit
```

Comment:
@fgeorgatos we dont need `CONFIGS_REPO` this is not part of `settings.yml` it is done by framework and accessed via `BUILDTEST_ROOT`/toolkit. 

This is assumed user sources the `sourceme.sh` provided by buildtest